templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -3,9 +3,19 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/keepalived.conf.tmpl"
 contents:
   inline: |
     vrrp_script chk_ocp {
-        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:6443/readyz && /usr/bin/curl -o /dev/null -kLfs http://localhost:50936/readyz"
+        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:9443/readyz"
         interval 1
-        weight 50
+        weight 6
+        rise 3
+        fall 2
+    }
+
+    vrrp_script chk_haproxy {
+        script "/usr/bin/curl -o /dev/null -kLfs http://localhost:50936/haproxyready"
```

Comment:
When does /haproxyready report failure? When all the servers in the master backend are down?

Reply:
To complete this change I think we should : 
A. Handle the firewall rule add/delete by Keepalived instead of haproxy-monitor
B. Haproxy-monitor should be responsible only for generating/updating haproxy configuration.

I think that the reason we see [1] in keepalived logs is the late start of HAProxy LB, it takes 3*6 seconds for haproxy-monitor to render the configuration.    
Maybe we should also change the monitor polling timing. 

[1] /usr/bin/curl -o /dev/null -kLfs https://localhost:6443/readyz && /usr/bin/curl -o /dev/null -kLfs http://localhost:50936/readyz exited with status 7

Reply:
I discussed this with Toni a bit yesterday and we realized that we actually need to keep the monitor check for the case where all loadbalancers go down, but at least one of the api servers is still up. If we just rely on keepalived to move the VIP then that wouldn't work because there's no functional haproxy for it to move to.